sql/parser/token.go

```diff
@@ -354,12 +360,14 @@ func (tkn *tokenizer) scanNumber(seenDecimalPoint bool) (int, []byte) {
 
 fraction:
```

Comment:
Well, we rely on `strconv.ParseFloat` for the actual conversion of the string to a `float64`, so octal floats aren't actually supported. I'll add a TODO to this code to clean it up.